packages/merkle-trees/src/lib.nr

```diff
@@ -57,6 +66,27 @@ trait MT_Creator {
     }
 }
 
+trait IMT_Creator {
+    fn default(root: Field, hasher: fn([Field; 2]) -> Field) -> Self;
+
+    /**
+     * Imports an existing Incremental Merkle Tree (IncrementalMerkleTree) instance.
+     * @param hasher The hash function that is used to hash the nodes of the tree
+     * @param root The root of the tree
+     */
+    fn from(root: Field, hasher: fn([Field; 2]) -> Field) -> Self {
+        Self::default(root, hasher)
+    }
+
+    /**
+     * Creates a new Incremental Merkle Tree (IncrementalMerkleTree) instance.
+     * @param hasher The hash function that is used to hash the nodes of the tree
+     */
+    fn new(hasher: fn([Field; 2]) -> Field) -> Self {
+        Self::from(0, hasher)
+    }
+}
+
```

Comment:
This is also the same as the existing `trait MT_Creator`. I guess you can re-use the same code.